packages/vuetify/cypress/support/mount.ts

```diff
@@ -1,5 +1,6 @@
 import { mount as cyMount } from '@cypress/vue'
 import { createVuetify } from '../../src/entry-bundler'
+import { mergeDeep } from '../../src/util'
```

Comment:
Should these import from `@/entry-bundler` and `@/util`?

Reply:
iirc vscode complained so I just let it be